### PR TITLE
Add identity IT

### PR DESCRIPTION
**Pull request by Zelldon**
Integrate in the current IT test identity, to verify whether the services are coming up and we can log in to identity/keycloak. This is later also necessary to access operate and tasklist.

Details:

 *  In order to log in we need to port-forward to identity (:8080) and keycloak (:18080)
 * We need to get the log in URL from identity (GET /auth/login to resolve url with session code)
 * Login with session URL and demo:demo user
 * After log in we get a JWT, which we verify on a identity resource
 
Since all of this can be quite flaky, due to the properties of distributed systems (many dependency's and fragile environment) we retry the log in.

Dear reviewers:

@oleschoenburg just check whether you understand what is happening. I will later do some more refactoring to split the functions in separate files etc.
@dlavrenuek just check whether it makes sense how we do the log in (it should be like we discussed)
@menski no need to review it, just for you if you want to check it out (since you were interested)

### CODE REVIEW

File: charts/ccsm-helm/charts/identity/templates/deployment.yaml [hunk 1]
```diff
@@ -20,6 +20,10 @@ spec:
         image: "{{ .Values.image.repository }}:{{ .Values.global.image.tag }}"
         {{- end }}
         env:
+          - name: KEYCLOAK_INIT_OPERATE_SECRET
+            value: "the-cake-is-alive"
+          - name: KEYCLOAK_INIT_OPERATE_ROOT_URL
+            value: "http://localhost:8080"
```
Comment (Zelldon) [inline charts/ccsm-helm/charts/identity/templates/deployment.yaml:26]: @dlavrenuek without this I can't log in is this correct?

Comment (dlavrenuek) [reply charts/ccsm-helm/charts/identity/templates/deployment.yaml:26]: Without this part users will not be able to login in Operate. If Operate is not configured to use Identity yet, these values can be removed for now.

Comment (Zelldon) [reply charts/ccsm-helm/charts/identity/templates/deployment.yaml:26]: Do you know why then log in didnt work without this?

Comment (dlavrenuek) [reply charts/ccsm-helm/charts/identity/templates/deployment.yaml:26]: What login do you mean? Login into Operate or Identity? Locally or in the tests?

Comment (Zelldon) [reply charts/ccsm-helm/charts/identity/templates/deployment.yaml:26]: The IT test failed with demo:demo without this setting. But i can also try it again when Im back

Comment (Zelldon) [reply charts/ccsm-helm/charts/identity/templates/deployment.yaml:26]: So not 100% true, what fails is the login with the jwt token.

**Comment by Zelldon** [comment]
Based on https://github.com/camunda/camunda-platform-helm/pull/263 and @dlavrenuek we found the issue with the Identity tests, we used before a role (operate) which didn't exist if operate is not configured. This is fixed now. I will go ahead and merge this.

**Pull request merged**